lib/media/ttml_text_parser.js

```diff
@@ -217,6 +208,25 @@ shaka.media.TtmlTextParser.getLeafNodes_ = function(element) {
 
 
 /**
+ * Insert \n where <br> tags are found
+ *
+ * @param {!Node} element
```

Comment:
No, childNodes is correct.  children is not supported in all browsers, which is why we use childNodes everywhere currently.